src/apps/views/AppsList/AppsList.tsx

```diff
@@ -52,27 +66,64 @@ export const AppsList: React.FC<AppsListProps> = ({ params }) => {
       field: AppSortField.CREATION_DATE
     }
   };
-  const [retryInstallApp, installRetryAppResult] = useAppRetryInstallMutation(
-    {}
-  );
+
+  const intervalId = useRef<null | number>(null);
+
+  const removeInstallation = (id: string) =>
+    setActiveInstallations(installations =>
+      installations.filter(item => item.id !== id)
+    );
+
   const {
     data: appsInProgressData,
     loading: loadingAppsInProgress,
     refetch: appsInProgressRefetch
   } = useAppsInProgressListQuery({
-    displayLoader: true,
+    displayLoader: false,
     variables: paginationState
   });
   const { data, loading, refetch } = useInstalledAppsListQuery({
     displayLoader: true,
     variables: queryVariables
   });
+
   const { loadNextPage, loadPreviousPage, pageInfo } = paginate(
     data?.apps?.pageInfo,
     paginationState,
     params
   );
-
+  const installedAppNotify = (name: string) => {
+    notify({
+      status: "success",
+      text: intl.formatMessage(
+        {
+          defaultMessage: "{name} is ready to be used",
+          description: "message content"
+        },
+        { name }
+      ),
+      title: intl.formatMessage({
+        defaultMessage: "App installed",
+        description: "message title"
+      })
+    });
+  };
+  const [retryInstallApp] = useAppRetryInstallMutation({
+    onCompleted: data => {
+      const errors = data.appRetryInstall.errors;
+      if (!errors.length) {
+        const appInstallation = data.appRetryInstall.appInstallation;
+        setActiveInstallations(installations => [
+          ...installations,
+          { id: appInstallation.id, name: appInstallation.appName }
+        ]);
+      } else {
+        errors.forEach(error =>
+          notify({ status: "error", text: error.message })
```

Comment:
These error messages should be translated like others, head to `src/utils/errors` for some examples